modules/ROOT/pages/configurable-archive.adoc

```diff
@@ -11,10 +11,10 @@ For Enterprise and Performance editions only.
 
 The Configurable Archive feature is provided by the ``com.bonitasoft.engine.archive.impl.ListArchivingStrategy ``service implementation, which only works with a *Enterprise or Performance license*.
 It enables you to specify for each FlowNode type whether or not to archive values when the flow node is changed. This determines what information
-(for example archived instances of processes and executed tasks) is available to an administrator in Bonita Portal.
+(for example archived instances of processes and executed tasks) is available to an administrator in Bonita Administrator Application.
 
 Deactivating archives has a positive effect on overall performance of the system as it reduces the number of entries inserted in the database. It has also the benefit of reducing the disk space used by the database.
-However, it prevents users from keeping a history of past activity. The most visible impact of this loss of information is through the Bonita Portal. Completed tasks will not be visible.
+However, it prevents users from keeping a history of past activity. The most visible impact of this loss of information is through the Bonita Applications. Completed tasks will not be visible.
 
 In general we recommended that you set all attributes to *true* or all to *false*. However, here are some details on some settings that you can change if users have a specific use case:
```

Comment:
In general we recommended that you set all attributes to *true* or all to *false*. => In general, we recommend you to set all attributes to *true* or all to *false*.

Reply:
Added